lua/lspconfig/server_configurations/sonarlint.lua

```diff
@@ -0,0 +1,92 @@
+local util = require 'lspconfig.util'
+
+local tcp = vim.loop.new_tcp()
+tcp:bind('127.0.0.1', 0)
```

Comment:
port in there is 0? no default port usage ?

Reply:
The whole snippet
```lua
local tcp = vim.loop.new_tcp()
tcp:bind('127.0.0.1', 0)
local port = tostring(tcp:getsockname().port
```
is to generate a random, unused port. The sonarlint language server doesn't have a default port.

Reply:
oh I ignored that code ..